### ISSUE TITLE
different mAP for ssd_mobilenet_v2_fpnlite_320x320_coco17 after training from scratch

### ISSUE THREAD

**Issue by altair199797** [tensorflow/models#10745]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [ y] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the documentation with the issue

https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md

## 2. Describe the issue

I have retrained the model with: 
https://github.com/tensorflow/models/blob/master/research/object_detection/configs/tf2/ssd_mobilenet_v2_fpnlite_320x320_coco17_tpu-8.config

However I can't reach more than a mAP of 19%, compared to the mAP of 22.2 % reported on the modelzoo.
I also trained on coco2017.
The config file from above features a initial learning rate of 0.08, could that be too low? Or are the number of steps too few?
Because in the config file the fine_tune_checkpoint attribute is still present.

**Comment by dsnsabari** [comment]
@tombstone, any updates on this issue?
